daphne_service_utils/src/config.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2024 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+use daphne::{
+    hpke::{HpkeConfig, HpkeReceiverConfig},
+    DapGlobalConfig, DapVersion,
+};
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+use crate::{auth::DaphneWorkerAuthMethod, DapRole};
+
+/// draft-wang-ppm-dap-taskprov: Long-lived parameters for the taskprov extension.
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct TaskprovConfig {
+    /// HPKE collector configuration for all taskprov tasks.
+    pub hpke_collector_config: HpkeConfig,
+
+    /// VDAF verify key init secret, used to generate the VDAF verification key for a taskprov task.
+    #[serde(with = "hex")]
+    pub vdaf_verify_key_init: [u8; 32],
+
+    /// Leader, Helper: Method for authorizing Leader requests.
+    pub leader_auth: DaphneWorkerAuthMethod,
+
+    /// Leader: Method for authorizing Collector requests.
+    pub collector_auth: Option<DaphneWorkerAuthMethod>,
+}
+
+pub type HpkeRecieverConfigList = Vec<HpkeReceiverConfig>;
+
+/// Daphne-Worker configuration, including long-lived parameters used across DAP tasks.
```

Comment:
Perhaps we want a generic term that applies to either daphne_worker or daphne_server
```suggestion
/// Daphne server configuration, including long-lived parameters used across DAP tasks.
```